src/Message/ButtonReply/ButtonAction.php

```diff
@@ -27,4 +34,25 @@ public function buttons(): array
 
         return $buttonActions;
     }
+
+    /**
+     * Get the header of the button action, if it exists
+     *
+     * @return array|null The header of the button action, or null if it doesn't exist
+     */
+    public function header(): array
+    {
+        $header = [];
```

Comment:
please, if you expect that header will be always an array so it's better that you don't define $header as nullable value. You can set as default value an empty array in the constructor and remove the nullable flag.

Reply:
Change reverted. It was to support media header for button reply but was breaking